tests/integration/devfile/cmd_devfile_log_test.go

```diff
@@ -24,51 +24,56 @@ var _ = Describe("odo devfile log command tests", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
-	Context("Verify odo log for devfile works", func() {
-
-		It("should log run command output and fail for debug command", func() {
-
+	When("When a springboot component is created", func() {
```

Comment:
Does it have to be a springboot component? Can it not be nodejs, so that we can combine the tests?

Reply:
It does not necessarily need to be springboot, however by using springboot we cover more than just nodejs. So I will keep springboot.